src/lookup-table/databaseMethods.ts

```diff
@@ -21,6 +20,41 @@ const databaseMethods = {
       throw err
     }
   },
+  getAllLookupTableStructures: async () => {
+    const result = await DBConnect.gqlQuery(
+      `query getAllLookupTableStructures {
+        dataTables(condition: {isLookupTable: true}) {
+          nodes {
+            id
+            tableName
+            displayName
+            fieldMap
+            __typename
+            dataViewCode
+          }
+          __typename
+        }
+      }
+    `
+    )
+    return result?.dataTables?.nodes
+  },
+  getLookupTableStructure: async (id: number) => {
+    const result = await DBConnect.gqlQuery(
+      `query getTableStructure($id: Int!) {
+        dataTable(id: $id) {
+          dataViewCode
+          displayName
+          fieldMap
+          id
+          tableName
+        }
+      }
+    `,
+      { id }
+    )
+    return result?.dataTable
+  },
```

Comment:
These are the same queries that used to be made directly by the front-end, but now put behind a REST endpoint.